controllers/service_controller.go

```diff
@@ -264,3 +278,17 @@ func (r *ServiceReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		For(&configv1beta1.Service{}).
 		Complete(r)
 }
+
+func (r *ServiceReconciler) getClusterSecrets(ctx context.Context, apiKeyConf *authorinoIdentity.APIKey) ([]string, error) {
```

Comment:
I could place this function within the ApiKey module in https://github.com/3scale-labs/authorino/pull/55/files#diff-25d3e852acdfdbd26bb9a6f3f0e2941b2a54b32d508483b3054790187e2079ac , maybe would be nicely encapsulated and tested within that package. WDYT @guicassolato ?

Reply:
Not sure if the APIKey identity should be aware of Secrets. For all purposes, the APIKey identity needs a list of authorized identities. If that list comes from Secrets, that's a details that can be abstracted IMO. In fact, even the `SetSecrets` function (as well as internal fields of the struct perhaps) should reflect that abstraction.